packages/providers/providers.yaml

```
@@ -4501,6 +4501,11 @@ google:
         access_type: offline
         prompt: consent
     proxy:
+        retry:
+            in_body:
+                path: error.message
+                value: User-rate limit exceeded.  Retry after ${value}
```

I feel like this prop could just be a regex instead of this custom syntax
```suggestion
                value: /User-rate limit exceeded.  Retry after (.+)/
```

---

updated to the new regex pattern matching logic in the latest commit  🚀